source/Playlist.h

```diff
@@ -0,0 +1,64 @@
+/* Playlist.h
+Copyright (c) 2022 by RisingLeaf
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
```

Comment:
#IFNDEF or whatever its called
How dare you forget it x)
Not saying it has to be on this line idk conventions by heart